XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/DefaultMixedRealityInputSystemProfile.asset

```diff
@@ -30,3 +29,5 @@ MonoBehaviour:
     type: 2}
   controllerVisualizationProfile: {fileID: 11400000, guid: d7daaacd4b9b4529a2bb2c06d10b5ae6,
     type: 2}
+  handTrackingProfile: {fileID: 11400000, guid: 5a440b753cec67d458def6f54cf9f705,
```

Comment:
Interesting thought but I don't think so. Hand tracking for me is definitely a specific form of input, kind of a controller. Similar to Oclus Touch / WMR controllers but instead of having one pointer, a hand actually kind of has 5 because I want to be able to use each finger to interact and select things. This is where the MRTK solution doesn't work for me. I always have to use the pointing finger.

Reply:
You are probably correct, so should it be a controller and not an alternate input system in that case?

Reply:
Basically yes. That's what I am working towards to currently. Basically I think it would make sense to have a controller mapping profile for "hand controllers" as well, just as with all other controllers. That way you could e.g. assign Actions per finger, if you consider each finger a "button on a controller". Does that make sense?

Reply:
It's certainly an option. But remember you shouldn't be forced to supply an action to a finger, just using the physical collision be the action (I push the button, the button clicks).  The important thing in that interaction is the light switch is flipped, but what finger did it.  But I can see both advantages.

Also to remember, fingers / hands shouldn't/don't need pointers to work. Physical interaction is enough for contact to contact.  Pointers would only be needed for far interaction from either the wrist or "pointing"/ "grabbing from afar"

Need to keep the physical goals in mind for how we humans actually use hands.

Make sense?

Reply:
Sure thing. The actions idea is just something that should be optional, if anything. But I guess if we really consider the hand to be a controller, it would make sense for consistency.

Thanks for the hint about pointers! Absolutely right and I've been focusing to much on those darn pointers lately.